src/patternfly/components/Select/select.scss

```diff
@@ -346,6 +354,14 @@
     right: 0;
   }
 
+  &.pf-m-wrap {
```

Comment:
Not sure if this is the intention but is `.pf-m-wrap` added to the parent `pf-c-select__menu` element to allow its children items to wrap? Which allows the description to wrap under the menu-item-title element? If so wouldn't that mean that if `.pf-m-wrap` is used and some of the items have descriptions and others don't, all of the items will now wrap?

Also, that would require the react component to go back up to the parent `__menu` to add a class when one of its children meets some condition, which isn't ideal.

What about just adding `.pf-m-description` to all items with descriptions (not just ones that are selected, as it is currently), and we add `white-space: normal;` to them, then add `.white-space: nowrap;` to the `__menu-item-title` element?

Otherwise this lgtm!